docs/DeveloperGuide.md

```diff
@@ -195,39 +195,39 @@ thrown if the person to add already exists in the contact list. Error message is
     * Cons: It is less flexible, since only female and male genders are accepted.
 
 * **Alternative 2:** Optional gender field:
-    * Pros: It is a more flexible implementation, since user has the choice to set gender to male or female, 
+    * Pros: It is a more flexible implementation, since user has the choice to set gender to male or female,
   as well as hide gender.
     * Cons: It is less logical since gender is usually a required binary field in most applications.
 
 ### Edit Gender
 
-The Edit Gender feature allows users to edit a gender field (Male / Female) of a person in the contact list. 
+The Edit Gender feature allows users to edit a gender field (Male / Female) of a person in the contact list.
 It is performed as a part of `editCommand#execute()`.
```

Comment:
Same as the `AddPersonCommand`, this line should be updated to match the diagram.

```suggestion
It is performed as a part of `editPersonCommand#execute()`.
```